packages/lib-classifier/src/plugins/tasks/experimental/highlighter/models/HighlighterAnnotation.js

```diff
@@ -0,0 +1,29 @@
+import { types } from 'mobx-state-tree'
+
+import Annotation from '@plugins/tasks/models/Annotation'
+import HighlighterLabel from './HighlighterLabel'
+
+const Highlight = types.model('Highlight', {
+  start: types.optional(types.number, 0),
+  end: types.optional(types.number, 0),
+  text: types.optional(types.string, ''),
+  labelInformation: types.optional(HighlighterLabel, { color: '', label: '' })
+})
+
+const Highlighter = types.model('Highlighter', {
+  taskType: types.literal('highlighter'),
+  value: types.optional(types.array(Highlight), [])
```

Comment:
```suggestion
  value: types.array(Highlight)
```

Tiny change here: `types.array` implements `types.optional` by default, so you only need to override that if you don't want the default empty array.